### ISSUE TITLE
Server won't start after system crash

### ISSUE THREAD

**Issue by bariumprof** [BevoTechPack/BevoTechPack#803]
While playing on the machine hosting my server, the computer crashed. I'm running the BevoTechPack mini. After restarting, the server won't start. Here's a link to the server launch log: https://www.dropbox.com/s/vni3x3urjnoa9va/fml-server-latest.log?dl=0

Hopefully someone can tell me how to get it up and running again?
Thanks!

**Comment by Xfel** [comment]
The log you posted doesn't really say that much, you should upload the server-log (should be named just "latest.log" and the crash report if you get one.

**Comment by bariumprof** [comment]
Alright, here's the link to that! Thanks for taking the time to look for me.
https://www.dropbox.com/s/4k6npyjuul5asav/latest.log?dl=0

**Comment by Xfel** [comment]
Um are you sure that this is the right log file? it doesn't really say as much as it should if it corresponds to the fml log file you posted earlier.

btw. I would recommend to post logs to pastebin instead of dropbox, so we can view them directly.

**Comment by bariumprof** [comment]
Yeah, that's definitely it. I noticed in the first log I posted, it says "The following problems were captured during this phase" and then doesn't list anything, like logs many other people post do. What would this mean?

I also reinstalled the modpack and copied the old world folder, but got the same log. Would that mean that the world itself is just corrupt? And if so, is there any way to fix that or am I just SOL here?
